pyproject.toml

```
@@ -100,7 +100,7 @@ version = "0.2.1"
 [tool.poetry.dependencies]
 aioarango = "^1.0.0"
 indexed = "^1.3.0"
-pydantic = "==1.10.12"
+pydantic = ">=1.10.17"
```

maybe we can already set this to 
```suggestion
pydantic = "<2"
```

---

The documentation claims that the `v1` modules were not added till 1.10.17. Another option is to update the imports to something like this:

```python
try:
    from pydantic.v1.fields import ModelField
except ImportError:
    from pydantic.fields import ModelField 
```